src/information_view/views.py

```diff
@@ -0,0 +1,19 @@
+from django.shortcuts import render, get_object_or_404
+from .models import Project, Developer
+from django.views import View
+
+
+# Create your views here.
+
+# Is log in required?
```

Comment:
Yes. Log in is required, according to our documentation.
However, as the implementation of the auth system is still in progress, you can ignore that right now.
I will add the authentication part later.